msteams-platform/bots/how-to/conversations/channel-messages-with-rsc.md

```diff
@@ -8,10 +8,7 @@ ms.localizationpriority: medium
 
 # Receive all channel messages with RSC
 
-> [!NOTE]
-> This feature is currently available in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md) only.
-
-The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is now extended to bot scenarios.
+The resource-specific consent (RSC) permissions model, originally developed for Teams Graph APIs, is extended to bot scenarios.
 
 Currently, bots can only receive user channel messages when they are @mentioned. Using RSC, you can now request team owners to consent for a bot to receive user messages across standard channels in a team without being @mentioned. This capability is enabled by specifying the `ChannelMessage.Read.Group` permission in the manifest of an RSC enabled Teams app. After configuration, team owners can grant consent during the app installation process.
```

Comment:
Let's delete this first sentence as it no longer holds now that this feature is being released

`Currently, bots can only receive user channel messages when they are @mentioned.`